packages/config-file/config-file.1.2.2/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "A library used to manage configuration files"
+description: """
+Config_file is a library used to manage the configuration file(s) of
+an application. You simply define your options and it performs the
+loading and saving of the options in a file. You can also use the same
+options in the command line.
+"""
+maintainer: ["Antonin Décimo <antonin@tarides.com>"]
+authors: [
+  "Antonin Décimo <antonin@tarides.com>"
+  "Fabrice Le Fessant <fabrice.lefessant@inria.fr>"
+  "Maxence Guesdon <maxence.guesdon@inria.fr>"
+  "Jean-Baptiste Rouquier <jrouquie@gmail.com>"
+]
+license: "LGPL-2.0-or-later"
+homepage: "https://github.com/MisterDA/config-file"
+bug-reports: "https://github.com/MisterDA/config-file/issues"
+depends: [
+  "dune" {>= "3.18"}
+  "ocaml" {>= "4.02"}
+  "camlp-streams"
+  "camlp4" {build}
+  "cppo" {>= "1.6.7" & build}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/MisterDA/config-file.git"
+x-maintenance-intent: ["(latest)"]
+url {
+  src:
+    "https://github.com/MisterDA/config-file/releases/download/release-1.2.2/config-file-release-1.2.2.tbz"
```

Comment:
the `dune-project` file still contains `(version release-1.2.2)` instead of `(version 1.2.2)`

Reply:
thank!